app/models/reaction.rb

```diff
@@ -13,11 +14,12 @@ class Reaction < ApplicationRecord
 
   counter_culture :reactable,
                   column_name: proc { |model|
-                    model.points.positive? ? "positive_reactions_count" : "reactions_count"
+                    PUBLIC_CATEGORIES.include?(model.category) ? "public_reactions_count" : "reactions_count"
                   }
   counter_culture :user
 
-  scope :positive, -> { where("points > ?", 0) }
+  scope :positive, -> { where(category: PUBLIC_CATEGORIES) }
+  scope :public_category, -> { where(category: PUBLIC_CATEGORIES) }
```

Comment:
Not my favorite method name: `.public_category` but I can't do `.public` because Ruby 😅